### PR TITLE
fix donate widget

### PR DESCRIPTION
**Pull request by Findus23**
- fixes the recurring donation widget (`on0` has to be `Piwik Supporter`)
- doesn't allow to modify the redirected URL
- removes `bb2_screener_` parameter. I am pretty sure it doesn't do anything.
- fix image breaking into the next line when selecting `$120/yr.`

### CODE REVIEW

File: plugins/CoreHome/Controller.php [hunk 1]
```diff
@@ -277,8 +277,21 @@ public function redirectToPaypal()
                 unset($parameters[$name]);
             }
         }
+        $paypalParameters = [
+            "cmd" => "_s-xclick"
+        ];
+        if (empty($parameters["onetime"]) || $parameters["onetime"] != "true") {
+            $paypalParameters["hosted_button_id"] = "DVKLY73RS7JTE";
+            $paypalParameters["currency_code"] = "USD";
+            $paypalParameters["on0"] = "Piwik Supporter";
```
Comment (tsteur) [inline plugins/CoreHome/Controller.php]: do we need to change something in paypal setting to make it work for Matomo Supporter?

Comment (Findus23) [reply plugins/CoreHome/Controller.php]: Probably. Using `Matomo Supporter` just shows a Paypal error. I guess this is the name of the paypal "product".

Comment (tsteur) [reply plugins/CoreHome/Controller.php]: @mattab could you add this product to PayPal or configure it somehow?

**Comment by Findus23** [comment]
As this has been merged, I guess we should move it from 3.9.0 to 3.8.0

**Pull request merged**